tests/test_matprod.py

```diff
@@ -61,6 +94,18 @@ def test_matprod(nfeed, antpairs, precision, method):
     if method.startswith("GPU"):
         true = true.get()
 
+    print(np.shape(out))
+    print(np.shape(true))
+    print(out)
+    print(true)
+    print(out - true)
+
     assert out.dtype.type == ctype
     assert out.shape == (obj.npairs, nfeed, nfeed)
     np.testing.assert_allclose(out, true)
+
+
+# antpairs = np.array([(1,2),(1,3),(3,2),(3,4),(4,4)])
+
+# test_matprod(2,True,True,1,"CPUMatChunk")
+test_matprod(2, True, True, 1, "GPUMatChunk")
```

Comment:
you don't need this statement. Just run `pytest` from the top-level directory to run tests.

Reply:
got it.